.github/bin/run-clang-tidy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -u
+
+TMPDIR="${TMPDIR:-/tmp}"
+
+readonly PARALLEL_COUNT=$(nproc)
+readonly FILES_PER_INVOCATION=5
+
+readonly FILES_TO_PROCESS=${TMPDIR}/clang-tidy-files.list
+readonly TIDY_OUT=${TMPDIR}/clang-tidy.out
+
+# Using clang-tidy-11 as it is the last one still checking for
+# google-runtime-references, non-const references in parameters - which is
+# IMO better than the relaxed Google styleguide rules later.
```

Comment:
Can you attribute this opinion to the correct username?

Reply:
Done.